### PR TITLE
Fixed #1759

### LINKED ISSUE #1759: Database exports not detected
### Subject of the issue
Database exports not detected.

### Your environment
* Version of WPScan: 3.8.22
* Version of Ruby: 2.6.8p205
* Operating System (OS): macOS

### Steps to reproduce
I've created a test environment with some mocked vulnerabilities to test.
Two database export exists in the website, located in the following paths:
- https://wordpress.vulnerable.arall.net/wp-content/uploads/dump.sql
- https://wordpress.vulnerable.arall.net/backups/wordpress.vulnerable.arall.zip

But when I run:
```
wpscan --url https://wordpress.vulnerable.arall.net --enumerate dbe
```
It only reports the last one:

```
[!] https://wordpress.vulnerable.arall.net/backups/wordpress.vulnerable.arall.zip
 | Found By: Direct Access (Aggressive Detection)
```

### Expected behavior
WPScan should report **both** database exports.

### Actual behavior
WPScan **only** reports **one** database export.

### What have you already tried
I monitor the HTTP calls with a proxy, and both files are reached.

What I think the issue is that WPScan is sending a header with `Range: bytes=0-3000`, and expect a `200` response. 
However, the server is responding with `206 Partial Content`.

### PR DESCRIPTION
**Pull request by 0n1shi**
#1759

### CODE REVIEW

File: app/finders/db_exports/known_locations.rb [hunk 1]
```diff
@@ -17,7 +21,7 @@ class KnownLocations < CMSScanner::Finders::Finder
         def aggressive(opts = {})
           found = []
 
-          enumerate(potential_urls(opts), opts.merge(check_full_response: 200)) do |res|
+          enumerate(potential_urls(opts), opts.merge(check_full_response: [200, 206])) do |res|
```
Comment (alexsanford) [inline app/finders/db_exports/known_locations.rb:24]: Could we use `valid_response_codes` here, rather than repeating the array?

**Review by alexsanford — COMMENTED** [review]
@0n1shi I just tested this, and it works well!

One small change request, and then we should be able to get this merged.

File: app/finders/db_exports/known_locations.rb (hunk 1 above)
Comment (0n1shi) [reply app/finders/db_exports/known_locations.rb:24]: @alexsanford 

Thank you for the review !
I thought I should follow the other finders but using `valid_response_codes` is much better, I will fix it :)

Comment (alexsanford) [reply app/finders/db_exports/known_locations.rb]: Great, thanks!

**Review by alexsanford — APPROVED** [review]
Tested, works fine for me. Will merge once the build finishes.

**Pull request merged**